sormas-ui/src/main/java/de/symeda/sormas/ui/contact/ContactDataForm.java

```diff
@@ -247,12 +250,7 @@ protected void addFields() {
 		FieldHelper.setReadOnlyWhen(getFieldGroup(), Arrays.asList(ContactDto.FOLLOW_UP_UNTIL), ContactDto.OVERWRITE_FOLLOW_UP_UTIL, Arrays.asList(Boolean.FALSE), false, true);
 		FieldHelper.setRequiredWhen(getFieldGroup(), ContactDto.OVERWRITE_FOLLOW_UP_UTIL, Arrays.asList(ContactDto.FOLLOW_UP_UNTIL), Arrays.asList(Boolean.TRUE));
 
-		for (Object propertyId : getFieldGroup().getBoundPropertyIds()) {
-			Field<?> field = getFieldGroup().getField(propertyId);
-			if (isFieldHiddenForCurrentCountry(propertyId)) {
-				field.setVisible(false);
-			}
-		}
+		initializeVisibilitiesAndAllowedVisibilities(null, viewMode);
```

Comment:
Also pass the disease to ContactDataForm and use it here; it doesn't actually make a difference right now, but it's better to do it properly right from the beginning (instead of potentially running into problems later).